trigger/patch_test.go

```diff
@@ -203,67 +182,6 @@ func (s *patchSuite) TestPatchOutcome() {
 	s.NotNil(n)
 }
 
-func (s *patchSuite) TestRunChildrenOnPatchOutcome() {
```

Comment:
Removing this test, because I feel like it would require mocking things in order to actually be useful. I think it's more painful than useful right now, as it breaks other tests and would require too much to fix.